### ISSUE TITLE
Using Pretrained Model on Non-GPU Devices

### ISSUE THREAD

**Issue by aayush1205** [TreB1eN/InsightFace_Pytorch#102]
I have been trying to use the pre-trained model on a laptop with no GPU. Implies, I can't use CUDA. It raises a flag asking me to set map_location = 'cpu' in torch.load function. There are ample instances of the function. Where and what should be the change I should make?

**Comment by aayush1205** [comment]
adding the clause to the load function statement will do the job. Also, declaring use_cuda to false globally worked for me. Closing the issue.

**Comment by brealisty** [comment]
> adding the clause to the load function statement will do the job. Also, declaring use_cuda to false globally worked for me. Closing the issue.

I also met this issue,.
pthon: 3.6.0
model: model_mobilefacenet.pth
config: use_mobilefacenet = True.

and add code line about map_location='cpu', but get anthor error: "list index out of range".  any helps?

**Comment by brealisty** [comment]
> > adding the clause to the load function statement will do the job. Also, declaring use_cuda to false globally worked for me. Closing the issue.
> 
> I also met this issue,. pthon: 3.6.0 model: model_mobilefacenet.pth config: use_mobilefacenet = True.
> 
> and add code line about map_location='cpu', but get anthor error: "list index out of range". any helps?

after debuging, the reason is mtcnn don't detect face on facebank data. eg: mask face, picture rotate 90°.
